Makefile

```diff
@@ -873,10 +875,16 @@ ifeq ($(IS_CONFIGURED),yes)
 ifeq ($(ENABLE_VERBOSE),yes)
 	- $(RM_F) $(LIBFLAME_A_PATH)
 	- $(RM_F) $(LIBFLAME_SO_PATH)
+	- $(RM_F) $(AR_OBJ_LIST_FILE)
+	- $(RM_F) $(AR_OBJ_LIST_FILE).atmp
+	- $(RM_F) $(AR_OBJ_LIST_FILE).sotmp
 else
 	@echo "Removing libraries from $(BASE_LIB_PATH)"
 	@$(RM_F) $(LIBFLAME_A_PATH)
 	@$(RM_F) $(LIBFLAME_SO_PATH)
+	@$(RM_F) $(AR_OBJ_LIST_FILE)
```

Comment:
ar_obj_list was not being removed on a 'make clean'. When make was run again, the .so and .a were not being built correctly. A 'make distclean' would let a user build again, but they needed to run configure. So I moved these deletes to the 'cleanlib' rule and this seems to allow for rebuilding correctly now. (without having to rerun the configure)